articles/modules/ROOT/pages/protecting-against-cypher-injection.adoc

```diff
@@ -1,34 +1,34 @@
 = Protecting against Cypher injection
 :slug: protecting-against-cypher-injection
-:author: Andrew Bowman
-:neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4
+:author: Andrew Bowman, Gemma Lamont
+:neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4, 5.x
 :tags: cypher, security
 :category: security
 
 == What is Cypher injection?
 
 Cypher injection is a way for maliciously formatted input to jump out of its context, and by altering the query itself, hijack the query and perform unexpected operations on the database.
 
-This is a cousin to SQL injection, but affecting our Cypher query language.
+This is a cousin to SQL injection, but affects our Cypher query language.
 
-One of the best illustrations of injection attacks is from an XKCD comic featuring Little Bobby Tables:
+A great illustration demonstrating SQL injection is from the XKCD comic featuring Little Bobby Tables:
 
 [link=https://xkcd.com/327/]
 image::https://imgs.xkcd.com/comics/exploits_of_a_mom.png[Little Bobby Tables]
 
-The boy's mother named him `Robert'; DROP TABLE STUDENTS;--`, ensuring that if his name was appended into a SQL statement that
-didn't clean their inputs or protect against an injection attack, the quote would end the context of a quoted name, the semicolon would end that statement,
-the STUDENTS table would be dropped, and the `--` would turn the remaining part of the query into a comment, so the remaining query could be ignored and avoid any syntax errors.
+The comic shows a mother who has named her child `Robert'; DROP TABLE STUDENTS;--`, ensuring that if his name was appended to an SQL statement,
+through string concatenation with no protections against injection attacks, that the additional quote would close the opened quote from the application,
+the semicolon would finish the statement, and the `--` would comment out the rest of the query, so that syntax errors could be avoided.
 
-This syntax is of course specific to SQL. Given that Bobby's mother would likely have continued her crusade, she might have had another child specifically for targeting Cypher, used by the leading graph database Neo4j.
-Lets call him Little Robby Labels, to differentiate him from his older brother:
+Given that Bobby’s mother would likely have continued her crusade, her next child might be named to target Cypher, used by the leading graph database Neo4j.
+Let’s call him Little Robby Labels.
 
 `"Robby' WITH DISTINCT true as haxxored MATCH (s:Student) DETACH DELETE s //"`
```

Comment:
Not sure if the references to Robby are intentional or if it's a typo that lived on. The original comic is about Bobby, and we reference Bobby quite a lot in this article too?